mods/vote/init.lua

```diff
@@ -337,7 +339,12 @@ if set == nil or minetest.is_yes(set) then
 						minetest.chat_send_all("Vote passed, " ..
 								#results.yes .. " to " .. #results.no .. ", " ..
 								self.name .. " will be kicked.")
-						minetest.kick_player(self.name, "The vote to kick you passed")
+						minetest.kick_player(self.name, "The vote to kick you passed. You will be temporarily banned for 10 minutes")
```

Comment:
```suggestion
						minetest.kick_player(self.name, "The vote to kick you passed. You have been temporarily banned for 10 minutes")
```

Note that by the time the player sees this message, they're aleady banned.